devito/operator/operator.py

```diff
@@ -939,21 +955,23 @@ def _emit_apply_profiling(self, args):
             if metrics:
                 perf("Global performance: [%s]" % ', '.join(metrics))
 
+            # Same as above, but excluding the setup phase, e.g. the CPU-GPU
+            # data transfers in the case of a GPU run, mallocs, frees, etc.
+            metrics = []
+
+            v = summary.globals.get('fdlike-nosetup')
+            if v is not None:
+                metrics.append("%.2f s" % fround(v.time))
+                metrics.append("%.2f GPts/s" % fround(v.gpointss))
+
+                perf("Global performance <w/o setup>: [%s]" % ', '.join(metrics))
```

Comment:
I tried to run locally the PR and I do not see any diff in the log, should I ?

Reply:
it should yes according to these changes